Remove invalid comma in ASA code sample

tomasr opened the pull request:
Fix extra comma in code sample (`INNER JOIN, impressions` -> `INNER JOIN impressions`)

azurecla commented:
Hi __@elfisher__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (elfish). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

tomasr committed:
Remove invalid comma in code sample

Fix extra comma in code sample (`INNER JOIN,` -> `INNER JOIN`)

tomasr committed:
Qualify table in 2 sample queries

Add change suggested by @JSeb225, and also fix next query over that was refering to `clocks` table rather than `clicks` table
